src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -147,25 +213,28 @@ class ValidationStep extends React.Component {
                                 placeholder="1.52"
                                 keyboardType="number-pad"
                                 value={this.state.amount1}
-                                onChangeText={amount1 => this.setValue({amount1})}
+                                onChangeText={amount1 => this.clearErrorAndSetValue('amount1', amount1)}
+                                errorText={this.getErrorText('amount1')}
                             />
                             <ExpensiTextInput
                                 containerStyles={[styles.mb1]}
                                 placeholder="1.53"
                                 keyboardType="number-pad"
                                 value={this.state.amount2}
-                                onChangeText={amount2 => this.setValue({amount2})}
+                                onChangeText={amount2 => this.clearErrorAndSetValue('amount2', amount2)}
+                                errorText={this.getErrorText('amount2')}
                             />
                             <ExpensiTextInput
                                 containerStyles={[styles.mb1]}
                                 placeholder="1.54"
                                 keyboardType="number-pad"
                                 value={this.state.amount3}
-                                onChangeText={amount3 => this.setValue({amount3})}
+                                onChangeText={amount3 => this.clearErrorAndSetValue('amount3', amount3)}
+                                errorText={this.getErrorText('amount3')}
                             />
-                            {!_.isEmpty(errorMessage) && (
```

Comment:
Rather than show all server errors inline (they are already shown in a modal anyway) I updated this so that we are only showing the "max attempts reached" message. It's not quite a "server error" since we show it when the view opens and not after we press the submit button.

In this case, maybe we should **stop** showing the inputs because there's no further action you could take at this step? Unsure if this came up in testing @kevinksullivan, but seems like it will create issues for "enable Save & Continue".

Here's what this looks like now:

![2021-09-14_15-01-39](https://user-images.githubusercontent.com/32969087/133353542-91ada343-8e39-492d-b234-f7daf3aa5521.png)

Unsure if there is some better place for this message.